downstream/modules/platform/proc-verify-network-connectivity.adoc

```diff
@@ -0,0 +1,52 @@
+[id="verify-network-connectivity_{context}"]
+
+= Verifying network connectivity
+
+[role=_abstract]
+
+To ensure successful migration of your data, verify that you have network connectivity from your new operator deployment to your old deployment database.
+
+.Prerequisites
+Take note of the host and port information from your existing deployment. This information is located in the postgres.py file located in the conf.d directory.
+
+.Procedure
+
+. Create a yaml file to verify the connection between your new deployment and your old deployment database:
++
+-----
```

Comment:
This yaml has some errors, can you replace it with the following?

```
---
apiVersion: v1
kind: Pod
metadata:
    name: dbchecker
spec:
  containers:
    - name: dbchecker
      image: registry.redhat.io/rhel8/postgresql-12:latest
      command: ["sleep"]
      args: ["600"]
```

Note, I also removed the namespace line so that it will just default to the namespace the user is in, removing the need for the customer to modify the yaml before applying it.

Reply:
Suggested changes complete.